src/iteminfo_query.cpp

```diff
@@ -0,0 +1,49 @@
+#include "iteminfo_query.h"
+
+#include <vector>
+#include <string>
+
+
+iteminfo_query::iteminfo_query() : iteminfo_query_base()
+{
+};
+
+iteminfo_query::iteminfo_query( const std::string &bits ) : iteminfo_query_base( bits )
+{
+};
```

Comment:
The semicolon after a function definition is not needed (not even for constructors). You don't have it after the other function definitions below either.

Reply:
👍